content/blog/10-things-you-can-do-with-neo/index.md

```diff
@@ -0,0 +1,192 @@
+---
+title: "10 Things You Can Do With Pulumi Neo"
+date: 2025-10-06
+draft: false
+meta_desc: "Discover 10 concrete ways Pulumi Neo helps automate infrastructure tasks while keeping humans in control."
+meta_image: meta.png
+authors:
+    - meagan-cojocar
+tags:
+    - ai
+    - platform-engineering
+    - pulumi-neo
+    - aws
+
+social:
+    twitter: "10 concrete workflows you can use Pulumi Neo for: auto-fix AWS Config violations, upgrade Lambda runtimes across accounts, generate infrastructure templates, respond to CVEs. Neo generates PRs, not production changes."
```

Comment:
Similar to my title suggestion, maybe "... you can use our new infrastructure agent, Neo, for: ..."